docs/playbooks/sql/write.md

```diff
@@ -20,15 +20,15 @@ For these examples, let's assume we own a `my_table` with a schema of `id int, v
 
 ## Inserting data
 
-When you write data, it **can only correspond to a single table** in a single transaction. For example, the following query would fail since it touches two different tables (if submitted in one transaction):
+When you write data, it can touch one or more tables in a single transaction. For example, the following query is totally valid and will insert values into two different tables—assuming there aren't issues like invalid syntax that causes the inserts to fail:
```

Comment:
The way this is worded feels slightly confusing to me.  I'm not sure exactly how to word it, but there's a nuance here that relies on how the sql is being sent to the contract.
For example calling the contract with a single string containing the statement below, i.e. "INSERT INTO my_table(id) VALUES(1); INSERT INTO other_table(id) VALUES(2);", will **NOT** work.  
However sending two strings in a single transaction WILL work, e.g. "INSERT INTO my_table(id) VALUES(1);", "INSERT INTO other_table(id) VALUES(2);"

Example of using solidity:
```
// works
mutate(0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72, [
  { tableId: 123, statement: "INSERT INTO my_table_11155111_123 VALUES(1);" },
  { tableId: 124, statement: "INSERT INTO my_table_11155111_124 VALUES(2);" }
]);

// does not work
mutate(
  0xC18360217D8F7Ab5e7c516566761Ea12Ce7F9D72,
  "INSERT INTO my_table_11155111_123 VALUES(1); INSERT INTO other_table_11155111_124 VALUES(2);",
  123
);
```

Example using the JS SDK
```
// works
await db.batch([
  db.prepare("INSERT INTO my_table_11155111_123 VALUES(1);"),
  db.prepare("INSERT INTO my_table_11155111_124 VALUES(2);")
]);

// does not work
await db.prepare(
  "INSERT INTO my_table_11155111_123 VALUES(1); INSERT INTO other_table_11155111_124 VALUES(2);"
).all();
```

Reply:
Since the description is in regard to the SQL language, maybe we could introduce the idea that each string must only affect a single table but you can send as many strings as you like in a transaction (limit block size).
I'm not sure if the term "string" right in this context?

Reply:
Sorry this comment is getting quite long, but to add more potential confusion...
The CLI work to incorporate batching isn't done yet, but it _could(should?)_ allow users to do
things like:
```
 user$ tableland write "INSERT INTO my_table_11155111_123 VALUES(1); INSERT INTO other_table_11155111_124 VALUES(2);" --chain local-tableland --privateKey <privkey>

Reply:
Will the SDK throw an error on that prepare statement with a clear reason? "Multiple writes much use the batch method" or the like.
The smart contract can't do it, but how many people are calling the smart contract directly?

Reply:
Currently the SDK throws an error only if the user is doing `autoWait: true` because the error comes from the Validator when it tries to process the statement.  
You bring up a good point, and I think we would want to open a new issue for this, but we should be checking for this in the SDK instead of sending the transaction and letting the Validator handle it.

Reply:
The error wording is currently: "parsing query: queries are referencing two distinct tables: <table 1 name> <table 2 name>"

Reply:
I've opened an issue for it: https://linear.app/tableland/issue/JST-22

Not 100% sure I capture what you meant, so feel free to suggest changes to the ticket. It's in triage right now, since I'm not sure when we might want to do it.

Reply:
thanks for the input yall! i've adjusted the language and tried to use database vs. blockchain transactions, plus the "statement string" concept, to make this a bit more clear.